### PR TITLE
Add system requirements for JetBrains editors

### PR DESCRIPTION
**Pull request by fioan89**
[ch2111] warn users about low resources

The recommendation is to have at leat 8 Gb of RAM
and 4 cores available to run projector and IntelliJ

image

### CODE REVIEW

File: environments/editors.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ launch the editor from the dashboard.
 > If you need a valid license to run your IDE locally, you'll also need one to
 > run it in Coder.
 
+### System requirements
+
+For the best possible experience it is recommended to run the editor in an
+environment with:
+
+- at least 8 Gb of RAM
+- a CPU with at least 4 cores
```
Comment (jakelang) [inline environments/editors.md:58]: I think that different wording would be better. Maybe something like
```
- At least 4 CPU cores
```

File: environments/editors.md [hunk 2]
```diff
@@ -49,6 +49,14 @@ launch the editor from the dashboard.
 > If you need a valid license to run your IDE locally, you'll also need one to
 > run it in Coder.
 
+### System requirements
+
+For the best possible experience it is recommended to run the editor in an
+environment with:
+
+- at least 8 Gb of RAM
```
Comment (jakelang) [inline environments/editors.md:57]: use `8 GB` or `8GB` rather than `8 Gb`.

Comment (jakelang) [inline environments/editors.md:57]: Also, capitalize bullet points:
`- At least`
better than
`- at least`

File: environments/editors.md (hunk 1 above)
Comment (vapurrmaid) [reply environments/editors.md:58]: alternatively, we could say:

For the best possible experience it is recommended to run the editor in an
environment with at minimum the following resources:

-8GB RAM
- 4 CPU cores

**Review by kylecarbs — CHANGES_REQUESTED** [review]

File: environments/editors.md (hunk 1 above)
Comment (fioan89) [reply environments/editors.md]: Done.

File: environments/editors.md (hunk 2 above)
Comment (fioan89) [reply environments/editors.md]: Done

File: environments/editors.md [hunk 3]
```diff
@@ -49,6 +49,12 @@ launch the editor from the dashboard.
 > If you need a valid license to run your IDE locally, you'll also need one to
 > run it in Coder.
 
+### System requirements
```
[the remark this answers is not in the document]
Comment (fioan89) [reply environments/editors.md]: Done

**Review by jakelang — APPROVED** [review]

**Pull request merged**